en/asgardeo/docs/guides/authentication/mfa/add-iproov-login.md

```diff
@@ -0,0 +1,133 @@
+          <td>OAuth Username</td>
+          <td>Provide the OAuth Username from the service provider created in iProov.</td>
+      </tr>
+      <tr>
+          <td>OAuth Password</td>
+          <td>Provide the OAuth password from the service provider created in iProov.</td>
+      </tr>
+      <tr>
+          <td>API Key</td>
+          <td>Provide the API key obtained from the service provider created in iProov.</td>
+      </tr>
+      <tr>
+          <td>API Secret</td>
+          <td>Provide the API secret obtained from the service provider created in iProov.</td>
+      </tr>
+      <tr>
+          <td>Enable Progressive Enrollment</td>
+          <td>Enable this to allow users to enroll with iProov during login.</td>
+      </tr>
+    </table>
+
+## Enable iProov login
+
+::: info Before you begin
+You need to <a :href="$withBase('/guides/applications/')">register an application with Asgardeo</a>. You can register your own application or use one of the <a :href="$withBase('/get-started/try-samples/')">sample applications</a> provided.
+:::
+
+To enable iProov login for your application:
+1. On the Asgardeo Console, go to **Applications**.
+
+2. Go to the **Sign-in Method** tab of the application and add iProov login from your preferred editor:
+
+   ::: details Using the Classic Editor
+   - If you haven't already configured an authentication flow,
+     1. Click **Start with default configuration** to begin.
+     2. Add a second authentication step by clicking the **+** icon and add your iProov connection to this step.
+
+   - If you have an already customized login flow, you can add a second authentication step by clicking the **+** icon and add your iProov connection to this step.
+     <img :src="$withBase('/assets/img/guides/mfa/iproov/customize-steps.png')" alt="Customize login flow">
+
+   :::
+
+   ::: details Using the Visual Editor
+      To add iProov login using the Visual Editor:
+   
+      1. Switch to the **Visual Editor** tab. 
+   
+      2. Add a second authentication step by clicking the **+** icon and add your iProov connection to this step.
+         <img :src="$withBase('/assets/img/guides/mfa/iproov/iproov-login-flow-with-visual-editor.png')" alt="Customize login flow">
+
+3. Click **Update** to save the sign-in flow.
+
+## Try it out
+
+Follow the steps given below:
+
+1. Access the application URL.
+
+2. Click **Login** to open the Asgardeo login page.
+
+3. Complete the preceding authentication steps.
+
+4. Grant camera access for iProov to authenticate the user.
+   <img :src="$withBase('/assets/img/guides/mfa/iproov/grant-camera-access.png')" alt="Grant camera access for iProov">
+
+5. Click **Scan Face**.
+   <img :src="$withBase('/assets/img/guides/mfa/iproov/scan-face-with-iproov.png')" alt="Scan face with iProov">
+
+6. Complete the iProov authentication process by starting the face scan.
+   <img :src="$withBase('/assets/img/guides/mfa/iproov/initiate-scan-face.png')" alt="Initiate face scan">
```

Comment:
Shall we combine these two steps as:

5. Click **Scan Face with iProov**, and complete the iProov authentication process by starting the face scan.